framework/src/modules/chain/defaults/config.js

```diff
@@ -325,6 +341,10 @@ const DefaultConfig = {
 					properties: {
 						blackList: {
 							type: 'array',
+							items: {
+								type: 'string',
+								format: 'ipOrFQDN',
```

Comment:
I am not sure if that format is valid here. As in the list when you specify the domain name during the startup we translate it to the IP and in application life cycle we always keep the IP. So if we match it with the blackList, that list must contains only IPs. 

Please verify from the code usage for its format.